cmd/npmExecuteLint.go

```diff
@@ -186,3 +207,12 @@ func findEslintConfigs(utils lintUtils) []string {
 	}
 	return eslintConfigs
 }
+
+func prepareArgs(defaultArgs []string, outputFileNamePattern, outputFileName string) []string {
+	fmt.Printf("[MH] outputFileName:%s", outputFileName)
```

Comment:
do we need this [MH] here?

Reply:
of course not :-) , removed in the meantime.